### DISCUSSION TITLE
How to exclude data on dbtable level for dbdump

### DISCUSSION THREAD

**Discussion by MartinSzotkovski** [spatie/laravel-backup#1567] [Ideas] [answered]
Hey guys,
is it possible with this package natively to do a dump so that I exclude data only for some tables?
To be precise, for example I have 50 tables, I want the dump to contain 48 tables with data, and for 2 tables only the structure.
MySQL 8.0.
I couldn't find this anywhere in the docs, do you think it's worth a Pull Request?

**Comment by erikn69** [comment, marked as answer]
You must search/ask on dumper, not here, here only handles backup, options comes from the other package
https://github.com/spatie/db-dumper
Maybe with `add_extra_option`
